[2024-06-22T17:35:51.000+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-22T17:30:49.487007+00:00 [queued]>
[2024-06-22T17:35:51.004+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-22T17:30:49.487007+00:00 [queued]>
[2024-06-22T17:35:51.004+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-06-22T17:35:51.010+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2024-06-22 17:30:49.487007+00:00
[2024-06-22T17:35:51.016+0000] {standard_task_runner.py:57} INFO - Started process 38124 to run task
[2024-06-22T17:35:51.018+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'scheduled__2024-06-22T17:30:49.487007+00:00', '--job-id', '2041', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmp4dawbip1']
[2024-06-22T17:35:51.019+0000] {standard_task_runner.py:85} INFO - Job 2041: Subtask kafka_data_stream
[2024-06-22T17:35:51.033+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 2041 for task kafka_data_stream (DagRun for kafka_spark_dag with run_id or execution_date of 'scheduled__2024-06-22T17:30:49.487007+00:00' not found; 38124)
[2024-06-22T17:35:51.049+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
